VideoLocker/src/main/java/org/edx/mobile/module/db/impl/DbHelper.java

```diff
@@ -14,17 +20,18 @@
  *
  */
 class DbHelper extends SQLiteOpenHelper {
-
     private SQLiteDatabase sqliteDb;
+    private Context mContext;
```

Comment:
Do we need to use these 'm' and 's' field prefixes? We've not been using them previously, and I don't see any need for them, since IDEs highlight their scopes anyway.